src/site/content/en/blog/content-indexing-api/index.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Indexing your offline-capable pages
```

Comment:
```suggestion
title: Indexing your offline-capable pages with the Content Indexing API
```

Because this is the canonical post on this topic, we should have the name of the API in the title. I'm also pretty sure that it has a significant impact on SEO.